src/index.js

```diff
@@ -73,6 +73,31 @@ export default class LMIFY {
 
     return this.packageManager.install(packages)
   }
+
+  async uninstall(packages) {
+    if (!packages) {
+      return
+    }
+
+    if (!Array.isArray(packages)) {
+      packages = [packages]
+    }
+
+    if (!packages.length) {
+      return
+    }
+
+    const [grant] = await Promise.all([
+      this._grant(packages),
+      this.init()
+    ])
+
+    if (!grant) {
+      return
+    }
+
+    return this.packageManager.uninstall(packages)
```

Comment:
`install`/`uninstall` logic seems duplicate now. What do you think if we add `do(action, ...args)` to the `package-manager` and also main class? `uninstall(packages, options` calls `this.do('uninstall', ...args)`

Reply:
(useful for later actions like upgrade, verify)

Reply:
I don't always want to do the same thing with install and uninstall.
Implementing do in package-managers might be a good idea, but applying it to the main also seems to be blocking future extensibility.
For example, how about providing install / uninstall as a Surface API and preparing (private) _do internally?